Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ResourceString/ResourceStringXaml.bind

```diff
@@ -0,0 +1,67 @@
+<Page
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.SampleApp.SamplePages.ResourceString"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:core="using:Microsoft.Xaml.Interactions.Core"
+    xmlns:ui="using:Microsoft.Toolkit.Uwp.UI"
+    mc:Ignorable="d">
+    <Page.Resources>
+        <SolidColorBrush x:Key="BrushRed" Color="Red" />
+        <SolidColorBrush x:Key="BrushBlue" Color="Blue" />
+    </Page.Resources>
+    <Grid>
+        <StackPanel Spacing="8"
+                    HorizontalAlignment="Center"
+                    VerticalAlignment="Center">
+            <TextBlock Text="{ui:ResourceString Name=ResourceStringMarkupTitle}"
+               FontSize="30em"
```

Comment:
`em` isn't a valid font unit in WinUI. This will parse, but so will `30abcdef`.
Let's remove any uses of `em`.